src/main/java/org/codehaus/mojo/exec/ExecJavaMojo.java

```diff
@@ -206,6 +206,25 @@ public class ExecJavaMojo
     @Parameter
     private List<String> classpathFilenameExclusions;
 
+    /**
+     * Whether to try and prohibit the called Java program from terminating the JVM (and with it the whole Maven build)
+     * by calling {@link System#exit(int)}. When active, a special security manager will intercept those calls. In case
+     * of an exit code 0 (OK), it will simply log the fact that {@link System#exit(int)} was called. Otherwise, it will
+     * throw a {@link SystemExitException}, failing the Maven goal as if the called Java code itself had exited with an
+     * exception. This way, the error is propagated without terminating the whole Maven JVM. In previous versions, users
+     * had to use the {@code exec} instead of the {@code java} goal in such cases, which now with this option is no
+     * longer necessary.
+     * <p>
+     * <b>Caveat:</b> Since JDK 17, you need to explicitly allow security manager usage when using this option, e.g. by
+     * setting {@code -Djava.security.manager=allow} in {@code MAVEN_OPTS}. Otherwise, the JVM will throw an
+     * {@link UnsupportedOperationException} with a message like "The Security Manager is deprecated and will be removed
+     * in a future release".
+     *
+     * @since 3.1.2
```

Comment:
3.2.0 - new feature
Please also check how documentation will look

Reply:
Oh, OK. I do not know about your versioning approach, sorry. Should I force-push, or do you prefer an extra commit?

Is anything wrong with documentation formatting, or was that just a general hint?

Reply:
The Maven site output looks OK:

![image](https://github.com/mojohaus/exec-maven-plugin/assets/1537384/0c372fe9-6df8-444e-b0d0-2fd1f9366ae2)

Please also not that locally, I updated to "since 3.2.0" already.